src/mesh/PacketHistory.cpp

```diff
@@ -7,152 +7,314 @@
 #endif
 #include "Throttle.h"
 
-PacketHistory::PacketHistory()
+/** PacketHistory additional debug level selection */
+#define mx_PH_DEBUGLEVEL 0 // Set to 2 for debug messages, 1 for info messages, 0 for no debug messages
```

Comment:
Before merging this, I'd prefer to remove the additional debug selection for PacketHistory.